internal/yunikorn/stream.go

```diff
@@ -64,20 +64,16 @@ func (s *Service) processStreamResponse(ctx context.Context, response []byte) er
 		logger.Errorf("error recording event: %v", err)
 	}
 
-	if eventRecord.GetType() == si.EventRecord_APP {
-		printAppEvent(&eventRecord)
-	}
+	logger.Infow(
```

Comment:
Without the check for it being an App event, this can generate a huge amount of logging. I wonder if we should add a config parameter to allow filtering by event type, so we don't have all-or-nothing logging?

Reply:
I don't have an opinion on this, you know more on this topic

Reply:
Let's go with your change for now; I'll write a ticket to make filtering of event-logging configurable, but it doesn't need to be done now.